### ISSUE TITLE
asterisk18: missing modules

### ISSUE THREAD

**Issue by fktpa813** [Entware/entware#124]
I could not find any packages that contains the following modules for Asterisk 1.8:

- app_db.so
- func_enum.so
- func_env.so
- func_module.so

These modules include functions that I'm using in my dial plans. This is the only issue that prevents me from switching from outdated Optware to Entware.

Please add missing packages to the repository. Thank you!

P.S. I suppose Asterisk 11 misses these modules as well although I didn't check this.

**Comment by ryzhovau** [comment]
asterisk18 packaged exactly as in OpenWrt. I can [provide](http://files.ryzhov-al.ru/Temp/asterisk-1.8.32.2.tgz) all modules for the current build, but I see no reason to change current asterisk18 packages, sorry.

You can [add](https://github.com/openwrt/packages/issues) an Issue for OpenWrt guys. All OpenWrt changes will be reflected here with next sync.

**Comment by ryzhovau** [comment]
No response.

**Comment by fktpa813** [comment]
Sorry for delayed response... I submitted a [ticket at OpenWRT/telephony](https://github.com/openwrt/telephony/issues/15) project, and they just checked in the fix. Looks good in the Makefile - the packages are created for all missing modules.
In the meantime - thanks for the current modules! I added them after installing entware Asterisk 1.8, and confirmed that all works as expected (no extra missing dependencies etc).
